docs/en/patterns/navigation.md

```diff
@@ -36,12 +36,22 @@ set the layout. This pattern includes a tagline which sits next to the logo.
 
 <a href="https://vanilla-framework.github.io/vanilla-framework/examples/patterns/navigation/sidebar/"
     class="js-example">
-    View example of the sidebar navigation pattern
+View example of the sidebar navigation pattern
+</a>
+
+### Sub-navigation
+
+Sub-navigation drop-down menus can be added to the navigation by including one
+or more `p-subnav` components. Note that this functionality requires JavaScript.
```

Comment:
Can you use the Vanilla notification pattern here please.

Reply:
Ok, done and extended the content slightly